jmx_prometheus_javaagent/src/main/java/io/prometheus/jmx/JavaAgent.java

```diff
@@ -1,24 +1,121 @@
 package io.prometheus.jmx;
 
+import io.prometheus.client.CollectorRegistry;
 import io.prometheus.client.exporter.MetricsServlet;
 import io.prometheus.client.hotspot.DefaultExports;
+
+import java.io.File;
+import java.io.IOException;
 import java.lang.instrument.Instrumentation;
 import java.io.FileReader;
+import java.nio.file.FileSystem;
```

Comment:
I'm afraid this was only added in Java 7, so we can't use it. 2/3 of our users still use Java 6.